easybuild/easyconfigs/l/libcdms/libcdms-3.1.2-foss-2020a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'libcdms'
+version = '3.1.2'
+homepage = 'https://github.com/CDAT/libcdms/'
+description = """Climate Data Management System Library."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/CDAT/libcdms/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['2010ef24f8f8c14a6f997206ec342e6fed9703b81f8641d70bebc992a3855e8c']
+
+dependencies = [
+    ('netCDF', '4.7.4'),
+    ('JasPer', '2.0.14'),
+    ('g2clib', '1.6.0'),
+    ('libjpeg-turbo', '2.0.4'),
+    ('Python', '3.8.2'),
+    ('libdrs', '3.1.2'),
+]
+
+configopts = "--prefix=%(installdir)s --enable-drs=yes"
+configopts += '--enable-hdf=no --enable-pp=yes --enable-ql=no --enable-grib2 '
```

Comment:
We have `HDF` easyconfigs already, why not include that as a dep too?

Reply:
Yes, that is an old issue with the libcdms, libcdms + HDF4 doesn't compile - found the same problem in a mailing list from maybe 2014 and the code hasn't changed since.